scalding-core/src/main/scala/com/twitter/scalding/reducer_estimation/Common.scala

```diff
@@ -1,16 +1,16 @@
 package com.twitter.scalding.reducer_estimation
 
-import cascading.flow.{ FlowStep, Flow, FlowStepStrategy }
+import cascading.flow.{Flow, FlowStep, FlowStepStrategy}
+import cascading.tap.hadoop.Hfs
 import com.twitter.algebird.Monoid
-import com.twitter.scalding.{ StringUtility, Config }
-import cascading.tap.{ Tap, CompositeTap }
-import org.apache.hadoop.fs.Path
-import org.apache.hadoop.mapred.{ FileInputFormat, JobConf }
+import com.twitter.scalding.{Config, StringUtility}
```

Comment:
seems like the scalding setting is to include a space after the `{` and before `}`. You might want to update intellij.